src/bitcoin.ts

```diff
@@ -71,39 +106,49 @@ router.get('/transactions', async (ctx, _next) => {
     requestHandler = () => blockchainService.transactions();
   }
 
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.get('/version', async (ctx, _next) => {
   const requestHandler = () => blockchainService.getServiceVersion();
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.get('/fee/:blockchainTime', async (ctx, _next) => {
   const requestHandler = () => blockchainService.getNormalizedFee(ctx.params.blockchainTime);
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.post('/transactions', async (ctx, _next) => {
   const writeRequest = JSON.parse(ctx.body);
   const requestHandler = () => blockchainService.writeTransaction(writeRequest.anchorString, writeRequest.minimumFee);
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.post('/transactions/firstValid', async (ctx, _next) => {
   const transactionsObject = JSON.parse(ctx.body);
   const requestHandler = () => blockchainService.firstValidTransaction(transactionsObject.transactions);
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.get('/time', async (ctx, _next) => {
   const requestHandler = () => blockchainService.time();
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
 });
 
 router.get('/time/:hash', async (ctx, _next) => {
   const requestHandler = () => blockchainService.time(ctx.params.hash);
-  await handleRequestAndSetKoaResponse(requestHandler, ctx.response);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
+});
+
+router.get('/locks/:identifier', async (ctx, _next) => {
+  const requestHandler = () => blockchainService.getValueTimeLock(ctx.params.identifier);
+  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
+});
+
+router.get('/writerlock', async (ctx, _next) => {
+  const requestHandler = () => blockchainService.getActiveValueTimeLockForThisNode();
+  handleRequestAndSetKoaResponse(requestHandler, ctx.response);
```

Comment:
Consider doing the below instead of introducing another synchronous version of the method.
```suggestion
  const requestHandler = async () => blockchainService.getActiveValueTimeLockForThisNode();
  await handleRequestAndSetKoaResponseAsync(requestHandler, ctx.response);
```